.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,15 +12,22 @@ assignees: ""
 - [ ] I have read [docs.revenuecat.com](https://docs.revenuecat.com/)
 - [ ] I have searched for [existing Github issues](https://github.com/RevenueCat/purchases-flutter/issues)
 
-**Describe the bug**
-A clear and concise description of what the bug is. The more detail you can provide the faster our team will be able to triage and resolve the issue. **Do not remove any of the steps from the template below. If a step is not applicable to your issue, please leave that step empty.**
+# ‼️ Required data ‼️ #
+**Do not remove any of the steps from the template below. If a step is not applicable to your issue, please leave that step empty.**
 
-1. Environment
-   1. Output of `flutter doctor`
-   2. How widespread is the issue. Percentage of devices affected.
-2. [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue
-3. Steps to reproduce, with a description of expected vs. actual behavior
-4. **Other information** (e.g. stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc.)
+There are a lot of things that can contribute to things not working. Having a very basic understanding of your environment will help us understand your issue faster!
 
-**Additional context**
+## Environment ##
+- [ ] Output of `flutter doctor`
+- [ ] Version of `purchases-flutter`
+- [ ] Testing device version e.g.: iOS 15.5, Android API 30, etc.
+- [ ] How often the issue occurs- every one of your customers is impacted? Only in dev?
+- [ ] [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue
+- [ ] Steps to reproduce, with a description of expected vs. actual behavior
```

Comment:
I'm wondering whether making it a checklist like this or putting placeholders for the information we ask might be better... I think this is how we have it in most SDKs, so I'm ok with this though

Reply:
Placeholder might be better, but I'mma merge this for now 😄